packages/core/src/modules/credentials/formats/jsonld/JsonLdCredentialFormatService.ts

```diff
@@ -146,9 +141,7 @@ export class JsonLdCredentialFormatService extends CredentialFormatService<JsonL
       throw new AriesFrameworkError('Missing jsonld payload in createOffer')
     }
 
-    const jsonLdCredential = new JsonLdCredentialDetail(jsonLdFormat)
-
-    MessageValidator.validateSync(jsonLdCredential)
+    MessageValidator.validateSync(jsonLdFormat.credentialAsJson)
```

Comment:
Mike, made this comment multiple times already, You can't validate JSON. It needs to be a class instance!!

We can parse it as an `JsonLdCredentialDetail` and then validate....